### DISCUSSION TITLE
Maker Workshop Installation and Setup Tutorial (fan contribution)

### DISCUSSION THREAD

**Discussion by AJ-Smoothie** [thelastoutpostworkshop/arduino-maker-workshop#13] [General]
I would love to support this extension! However, my skills are not up to par to actually contribute to the code. I do like making tutorials though, so here's a link to a tutorial if you are having some issue getting this extension setup.

@thelastoutpostworkshop let me know if I can be of anymore help!

https://www.youtube.com/watch?v=7lA5iIJi72c&t=62s

**Comment by thelastoutpostworkshop** [comment]
That's great!
